src/test/resources/test-default-servers.yaml

```diff
@@ -0,0 +1,17 @@
+server:
+  type: default
+  applicationConnectors:
+    - type: http
+      port: 0
+  adminConnectors:
+    - type: http
+      port: 0
+swagger:
+  resourcePackage: in.vectorpro.dropwizard.swagger
+  servers:
+    - url: https://development.gigantic-server.com/v1
```

Comment:
Please update the URL values in server list with dummy example urls

**Exa:**
- https://development.example.com/
- https://staging.example.com/
- https://prod.example.com/

Reply:
Done!